barkochba/templates/barkochba/main.html

```diff
@@ -254,7 +264,7 @@
 			<div class="col-sm-4 barkochba-story-list-column">
 				<div class="list-group">
 					{% for story_map in stories %}
-						<a href="#" class="list-group-item barkochba-story-summary" id="story-summary-{{ story_map.story.id }}" data-person-ids="{{ story_map.person_ids_json }}" data-story-id="{{ story_map.story.id }}" onclick="_storySelected({{ story_map.story.id }});">
+						<a href="#" class="list-group-item barkochba-story-summary" id="story-summary-{{ story_map.story.id }}" data-story-id="{{ story_map.story.id }}" data-person-ids="{{ story_map.person_ids_json }}" data-original-person-ids="{{ story_map.person_ids_json }}" onclick="_storySelected({{ story_map.story.id }});">
```

Comment:
A data-person-ids-t használom arra, hogy megjegyezze azokat, amiket már beírt a user, akkor is ha elkattint másik barkochba-történethez. Nem tudom, hogy lehetne-e szebben.